DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: neo4r
 Title: A 'Neo4J' Driver
-Version: 0.1.3
+Version: 4.0.0
```

Comment:
I'm not sure we should switch from 0.1.3 to 4.0.0 when it comes to package version number. 

I'd like to follow semantic versioning, and for me this one will be `0.2.0`